S2VX.Game/Timeline.cs

```diff
@@ -134,6 +155,10 @@ protected override void Update()
             var songRatio = story.GameTime / story.Track.Length;
             var newX = songRatio * bar.DrawWidth;
             slider.X = (float)Math.Clamp(newX, 0, bar.DrawWidth);
+
+            TimeSpan time = TimeSpan.FromMilliseconds(Math.Clamp(story.GameTime, 0, story.Track.Length));
+            clock.Text = time.ToString(@"mm\:ss\:fff");
+            TextSize = story.DrawWidth / 40;
```

Comment:
Lol, this seems a bit awkward. Tbh, working more with osu! framework, specifically the UI parts, makes using relative units somewhat difficult. May want to re-evaluate how we think about relative vs absolute units.